src/Microsoft.Dnx.Compilation.Abstractions/ILibraryExporter.cs

```diff
@@ -8,12 +8,15 @@ namespace Microsoft.Dnx.Compilation
     /// </summary>
     public interface ILibraryExporter
     {
-        LibraryExport GetLibraryExport(string name);
```

Comment:
Prefer the old names. The exporter is providing exports, not actually exporting anything as a verb.